.azuredevops/modulePipelines/ms.app.managedenvironments.yml

```diff
@@ -0,0 +1,147 @@
+name: 'Container Apps: Managed Environments'
+
+on:
+  workflow_dispatch:
+    inputs:
+      removeDeployment:
+        type: boolean
+        description: 'Remove deployed module'
+        required: false
+        default: true
+      prerelease:
+        type: boolean
+        description: 'Publish prerelease module'
+        required: false
+        default: false
```

Comment:
This entire file is actually a GitHub workflow that should be in the `.github/workflows` folder. Likewise, the other file shuold be merged here